install/kubernetes/agent/argo-agent-agent-userpass.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-agent-agent-userpass
+stringData:
+  credentials: ""
```

Comment:
Likewise, I don't think this should be part of the installation manifests, since it contains eventually secret data (`credentials`) and eventually is managed by an external entity such as ESO.

Reply:
I understand, as eventually it will be managed by ESO it is good Idea to remove it. My Idea to have it was to have a list of resources that should be present for both components (agent and principal) and template of the secret. But it can be done by documentation and this can be avoided.
Same for the above comment.